pandas/tests/indexes/interval/test_base.py

```diff
@@ -12,7 +13,7 @@ class TestBase(Base):
     in test_interval.py or the specific test file (e.g. test_astype.py)
     """
 
-    _holder = IntervalIndex
+    _holder = IntervalIndex  # type: Optional[Type[IntervalIndex]]
```

Comment:
you can probably revert this change now.

Reply:
removed